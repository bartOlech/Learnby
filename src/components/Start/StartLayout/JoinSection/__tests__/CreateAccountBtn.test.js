import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import CreateAccoundBtn from '../CreateAccountBtn';
import { BrowserRouter as Router, Link } from 'react-router-dom';

test('render', () => {
    const { asFragment } = render(<Router><CreateAccoundBtn/></Router>)
    expect(asFragment()).toMatchSnapshot();
})

test('It should return a specific value', () => {
    const { getByText } = render(<Router><CreateAccoundBtn/></Router>) 
    expect(getByText('Załóż konto')).toHaveTextContent('Załóż konto')
})
