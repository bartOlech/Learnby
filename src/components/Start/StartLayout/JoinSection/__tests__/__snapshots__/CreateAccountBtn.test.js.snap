// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<DocumentFragment>
  <div
    class="sc-bdVaJa cBJcMp"
  >
    <a
      href="/login"
    >
      <button
        class="sc-bwzfXH kOWHHY"
      >
        Załóż konto
      </button>
    </a>
  </div>
</DocumentFragment>
`;
